### PR TITLE
Update lip 0063 according to Legacy module migration logic

### PR DESCRIPTION
**Pull request by sergeyshemyakov**

### CODE REVIEW

File: proposals/lip-0063.md [hunk 1]
```diff
@@ -332,24 +342,29 @@ In the following, let `accountsState` be the key-value store of the accounts sta
...
         userObj.address = account.address
         userObj.balance = account.token.balance
         legacyObj.accounts.append(userObj)
+        legacyReserveAmount += account.token.balance
     data = serialization of legacyObj using genesisLegacyStoreSchema
     append {"moduleID": MODULE_ID_LEGACY, "data": data} to b.assets
+    # mint all the tokens for legacy accounts here
+    Token.mint(ADDRESS_LEGACY_RESERVE, TOKEN_ID_LSK_MAINCHAIN, legacyReserveAmount)
+    Token.lock(ADDRESS_LEGACY_RESERVE, MODULE_ID_LEGACY, TOKEN_ID_LSK_MAINCHAIN, legacyReserveAmount)
 ```
```
Comment (MaximeGagnebin) [inline proposals/lip-0063.md:360]: I think this logic is not really following the migration paradigm.
The migration should not mint + lock tokens. This amount should be  added to the `assets` entry for the token module under the correct address.

**Review by MaximeGagnebin — CHANGES_REQUESTED** [review]
I think the logic for handling the amounts for legacy accounts should be revisited.

**Review by janhack — CHANGES_REQUESTED** [review]
Ah, I missed the context here. We are actually just computing the genesis block and therefore cannot call any APIs or make any state changes. As Maxime noted, we would need to modify the asset of the Token module instead.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Review by MaximeGagnebin — APPROVED** [review]
According to the current specs, I think the different asset arrays need to be sorted (each module defines the sorting to be respected)
I would approve the PR as I think the Legacy module change is now settled.

**Review by janhack — CHANGES_REQUESTED** [review]
I think there is an issue with the current specifications if `ADDRESS_LEGACY_RESERVE` has any balance.

**Review by janhack — APPROVED** [review]
Looks all good now!

**Review by karmacoma — APPROVED** [review]

**Pull request merged**